### PR TITLE
Add Giphy Integration to Clack

### PR DESCRIPTION
**Pull request by mhtien**
Wanted to be able to attach Gifs to my
messages, and using the new beta components allows
me to add this Giphy Integration!

Discussed with Adam about how to do this and we ended
up with adding an api route to get the api token
from the server and store this within a config context
(incase we have any more that we want to retrieve).

I've applied for a production key from Giphy
(currently we can make a max of 100 api calls per
hour) so hopefully we can get this soon.

Test Plan:
Add GIPHY_API_KEY in .env and ran Clack
locally. Can attach and send gifs!

### CODE REVIEW

File: src/client/components/ToolbarWithGiphy.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+      <StyledGiphyAttribution
+        src="/static/powered-by-giphy.png"
+        alt="Giphy attribution"
+      />
+    </>
+  );
+}
+
+// The &&& is to increase specificity
+const StyledGiphtMenuButton = styled(betaV2.Button).attrs({
```
Comment (dmmiller) [inline src/client/components/ToolbarWithGiphy.tsx:171]: misspelling : GiphT

**Review by dmmiller — APPROVED** [review]
must giphy all the things!

File: src/client/components/ToolbarWithGiphy.tsx (hunk 1 above)
Comment (lazybean) [reply src/client/components/ToolbarWithGiphy.tsx:171]: Giphy is a gift!

File: src/client/components/ToolbarWithGiphy.tsx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  className: 'cord-tertiary',
+})`
+  &&& {
+    padding: 4px;
```
Comment (lazybean) [inline src/client/components/ToolbarWithGiphy.tsx:175]: That much? That must be a bug in our component.

File: src/client/components/ToolbarWithGiphy.tsx (hunk 1 above)
Comment (lazybean) [reply src/client/components/ToolbarWithGiphy.tsx:171]: TIL `.attrs` 😮

Comment (lazybean) [reply src/client/components/ToolbarWithGiphy.tsx:171]: It it not removing other classnames?

File: src/client/components/ToolbarWithGiphy.tsx (hunk 2 above)
Comment (mhtien) [reply src/client/components/ToolbarWithGiphy.tsx:175]: It was too confusing to figure out which classes I needed to get this so I just added it 😅

File: src/client/components/ToolbarWithGiphy.tsx (hunk 1 above)
Comment (mhtien) [reply src/client/components/ToolbarWithGiphy.tsx:171]: Surprisingly no it just adds more on! 🎉

**Comment by jwatzman** [comment]
> Add GIPHY_API_KEY in .env and ran Clack locally.

Make sure to add to the `.env` in 1password if you haven't already! And we'll need to manually add it to the `.env` on the clack server. I think all the directions for ssh access are in Notion, and then you just edit the `.env` by hand, LMK if you need help with this (the clack deploy is wonky).

**Pull request merged**